### PR TITLE
[ENH] bambenek parser: add malware name from the url available in the feed

### PR DESCRIPTION
**Pull request by navtej**
Adds malware name from the last column (url with file named after malware) in the csv.

### CODE REVIEW

File: intelmq/bots/parsers/bambenek/parser.py [hunk 1]
```diff
@@ -11,6 +11,13 @@ class BambenekParserBot(ParserBot):
     DOMMASTERLIST = {'http://osint.bambenekconsulting.com/feeds/c2-dommasterlist.txt'}
     DGA_FEED = {'http://osint.bambenekconsulting.com/feeds/dga-feed.txt'}
 
+    MALWARE_NAME_MAP = {
+        "cryptolocker": "cl",
```
Comment (wagner-certat) [inline intelmq/bots/parsers/bambenek/parser.py]: Where do these mappings come from? "cl" is not a very descriptive malware name.

File: intelmq/bots/parsers/bambenek/parser.py [hunk 2]
```diff
@@ -23,6 +30,12 @@ def parse_line(self, line, report):
             event.add('event_description.url', value[3])
             event.add('raw', line)
 
+            # last row is a url with malware named txt file link
```
Comment (wagner-certat) [inline intelmq/bots/parsers/bambenek/parser.py]: Do you mean column instead of row here?

Comment (navtej) [inline intelmq/bots/parsers/bambenek/parser.py]: yes.

File: intelmq/bots/parsers/bambenek/parser.py (hunk 1 above)
Comment (navtej) [inline intelmq/bots/parsers/bambenek/parser.py]: my bad keys and values are switched here. sending out a fix.

File: intelmq/bots/parsers/bambenek/parser.py [hunk 3]
```diff
@@ -23,6 +32,12 @@ def parse_line(self, line, report):
             event.add('event_description.url', value[3])
             event.add('raw', line)
 
+            # last column is a url with malware named txt file link
+            malware_name = value[-1].split('/')[-1].split('.')[0]
+            if malware_name in BambenekParserBot.MALWARE_NAME_MAP:
```
Comment (wagner-certat) [inline intelmq/bots/parsers/bambenek/parser.py]: These three lines can be reduced to one using dict's getter and the default value:
```python
event.add('malware.name', MALWARE_NAME_MAP.get(malware_name, malware_name))
```

**Comment by wagner-certat** [comment]
Thanks!

Did it myself in f5a81ce3

**Comment by navtej** [comment]
Thanks I was abt to come to this one

**Pull request merged**